[AX]: check-webkit-style - resolve (almost) all remaining warnings

dusek opened the pull request:
#### 1cf8ec92362b6115f9d21b5a6d1a85fd30c2cfba
<pre>
[AX]: check-webkit-style - resolve (almost) all remaining warnings
<a href="https://bugs.webkit.org/show_bug.cgi?id=297230">https://bugs.webkit.org/show_bug.cgi?id=297230</a>

Reviewed by Tyler Wilcock.

Reduces number of warnings in accessibility implementation to just 12:
$ Tools/Scripts/check-webkit-style Source/WebCore/accessibility
[...]
Total errors found: 12 in 152 files

Further 2 will be fixed as part of <a href="https://github.com/WebKit/WebKit/pull/49218">https://github.com/WebKit/WebKit/pull/49218</a>
(and respective changes were removed from this commit just to avoid conflict).

The remaining 10 warnings feel like (or I found they actually are) bugs
in check-webkit-style (or code style documentation)  which I will report
as such (and through that might learn if they indeed are or if
accessibility code needs to be further fixed).

The PR (linked in bug) tries to provide diff comments for changes that
might not be obvious.

* Source/WebCore/accessibility/AXGeometryManager.cpp:
* Source/WebCore/accessibility/AXLogger.cpp:
(WebCore::operator&lt;&lt;):
* Source/WebCore/accessibility/AXObjectCache.cpp:
(WebCore::AXObjectCache::handleAttributeChange):
(WebCore::AXObjectCache::characterOffsetForNodeAndOffset):
(WebCore::AXObjectCache::textMarkerDataForVisiblePosition):
(WebCore::characterForCharacterOffset):
* Source/WebCore/accessibility/AXTextMarker.cpp:
(WebCore::AXTextMarker::lineRange const):
* Source/WebCore/accessibility/AXTextRun.cpp:
* Source/WebCore/accessibility/AccessibilityAttachment.cpp:
* Source/WebCore/accessibility/AccessibilityMathMLElement.h:
* Source/WebCore/accessibility/AccessibilityMockObject.cpp:
* Source/WebCore/accessibility/AccessibilityMockObject.h:
* Source/WebCore/accessibility/AccessibilityObject.cpp:
(WebCore::AccessibilityObject::leftLineVisiblePositionRange const):
(WebCore::AccessibilityObject::rightLineVisiblePositionRange const):
(WebCore::AccessibilityObject::scrollToMakeVisibleWithSubFocus const):
(WebCore::AccessibilityObject::scrollToGlobalPoint const):
(WebCore::AccessibilityObject::isWithinHiddenWebArea const):
(WebCore::AccessibilityObject::isIgnoredWithoutCache const):
* Source/WebCore/accessibility/AccessibilityProgressIndicator.cpp:
* Source/WebCore/accessibility/AccessibilityRenderObject.cpp:
(WebCore::AccessibilityRenderObject::anchorElement const):
* Source/WebCore/accessibility/AccessibilitySVGObject.cpp:
(WebCore::AccessibilitySVGObject::parentObject const):
* Source/WebCore/accessibility/AccessibilityTable.cpp:
* Source/WebCore/accessibility/AccessibilityTableRow.cpp:
(WebCore::AccessibilityTableRow::addChildren):
* Source/WebCore/accessibility/AccessibleSetValueEvent.cpp:
* Source/WebCore/accessibility/AccessibleSetValueEvent.h:
* Source/WebCore/accessibility/ForcedAccessibilityValue.h:
* Source/WebCore/accessibility/atspi/AccessibilityObjectCollectionAtspi.cpp:
(WebCore::AccessibilityObjectAtspi::matches):
* Source/WebCore/accessibility/ios/AXObjectCacheIOS.mm:
* Source/WebCore/accessibility/ios/AccessibilityObjectIOS.mm:
* Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.h:
* Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.mm:
* Source/WebCore/accessibility/isolatedtree/AXIsolatedTree.cpp:
(WebCore::AXIsolatedTree::applyPendingChangesLocked):
* Source/WebCore/accessibility/isolatedtree/mac/AXIsolatedObjectMac.mm:
* Source/WebCore/accessibility/mac/AXObjectCacheMac.mm:
(WebCore::exerciseIsIgnored):
* Source/WebCore/accessibility/mac/AccessibilityObjectMac.mm:
(WebCore::AccessibilityObject::convertRectToPlatformSpace const):
(WebCore::AccessibilityObject::subrolePlatformString const):
* Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperBase.mm:
(convertPathToScreenSpaceFunction):
(-[WebAccessibilityObjectWrapperBase accessibilityVisibleCharacterRange]):
(-[WebAccessibilityObjectWrapperBase lineRectsAndText]):
* Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm:
(-[WebAccessibilityObjectWrapper accessibilityActionNames]):
(-[WebAccessibilityObjectWrapper accessibilityAttributeNames]):
(-[WebAccessibilityObjectWrapper windowElement:]):
(-[WebAccessibilityObjectWrapper accessibilityAttributeValue:]):
(-[WebAccessibilityObjectWrapper _accessibilityHitTest:returnPlatformElements:]):
(-[WebAccessibilityObjectWrapper accessibilityIsAttributeSettable:]):
(-[WebAccessibilityObjectWrapper accessibilityIsIgnored]):
(-[WebAccessibilityObjectWrapper accessibilityParameterizedAttributeNames]):
(-[WebAccessibilityObjectWrapper accessibilityPerformAction:]):
(-[WebAccessibilityObjectWrapper accessibilitySetValue:forAttribute:]):
(-[WebAccessibilityObjectWrapper _accessibilitySetValue:forAttribute:]):
(-[WebAccessibilityObjectWrapper accessibilityActionDescription:]):
(-[WebAccessibilityObjectWrapper accessibilityAttributeValue:forParameter:]):
(-[WebAccessibilityObjectWrapper ALLOW_DEPRECATED_IMPLEMENTATIONS_END]): Deleted.
* Source/WebCore/accessibility/win/AccessibilityObjectWrapperWin.h:
(WebCore::AccessibilityObjectWrapper::attached const):
(WebCore::AccessibilityObjectWrapper::accessibilityObject const):
(WebCore::AccessibilityObjectWrapper::AccessibilityObjectWrapper):

Canonical link: <a href="https://commits.webkit.org/298549@main">https://commits.webkit.org/298549@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/9c7873c13eeaff9e8444f8b8be21c243cb19a527

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/115824 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/35478 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/26010 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/121871 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/66347 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/117713 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/36167 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/44065 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/87984 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/42607 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/118772 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/28857 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/103941 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/68389 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/27994 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/22051 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/65543 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/98249 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/22177 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/125024 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/42711 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/32046 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/96737 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/43077 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/100123 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/96523 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/24562 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/41785 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/19657 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/38633 "The change is no longer eligible for processing.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/42600 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/48189 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/42067 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/45401 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/43774 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

dusek commented on Source/WebCore/accessibility/ios/AccessibilityObjectIOS.mm:
```diff
@@ -355,8 +355,8 @@ static void attributedStringSetCompositionAttributes(NSMutableAttributedString *
 }
 
-}
+} // namespace Accessibility
```
This namespace comment on namespace closing brace makes `check-webkit-style` stop complaining about `whitespace/blank_line`.

dusek commented on Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.mm:
```diff
@@ -40,11 +40,11 @@
 #import "HTMLInputElement.h"
 #import "HTMLNames.h"
 #import "HTMLTextAreaElement.h"
+#import "HitTestResult.h"
```
Header sort must be case-sensitive (as `sort` would produce) – code style is [explicit](https://webkit.org/code-style-guidelines/#include-others) on that 🤷.

dusek commented on Source/WebCore/accessibility/mac/AccessibilityObjectMac.mm:
```diff
@@ -105,9 +105,11 @@
-ALLOW_DEPRECATED_DECLARATIONS_END
+        ALLOW_DEPRECATED_DECLARATIONS_END
+
         return NSRectToCGRect(nsRect);
```
Extra newlines before and after added just for clarity after `BEGIN`/`END` macros are no longer indented to fix indentation warning (otherwise they would blend with other code, which was not the case before).

dusek commented on Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperBase.mm:
```diff
@@ -458,22 +459,19 @@ static void convertPathToScreenSpaceFunction(PathConversionInfo& conversion, con
         CGPathMoveToPoint(newPath, nil, newPoint.x, newPoint.y);
         break;
     }
-    case PathElement::Type::AddLineToPoint:
```
The other option would be:
```cpp
case PathElement::Type::MoveToPoint:
    {
        rect = FloatRect(element.points[0], FloatSize());
        CGPoint newPoint = [wrapper convertRectToSpace:rect space:AccessibilityConversionSpace::Screen].origin;
        CGPathMoveToPoint(newPath, nil, newPoint.x, newPoint.y);
        break;
    }
case ...
```

That would work a bit better at the end of the switch statement (avoid two closing braces indended at same level – see below), but would add more indentation, plus would be much larger diff in the PR. Also I found it used less in the code base (though it's used sometimes) than what I went for.

For all those reasons, I settled for the above, but happy to change if other form is preferred.

twilco replied:
What you have now is what I've always seen in other areas, so let's keep this 👍

dusek commented on Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm:
```diff
@@ -2185,10 +2183,10 @@ - (BOOL)accessibilityIsAttributeSettable:(NSString*)attributeName
 
     return NO;
 }
+ALLOW_DEPRECATED_IMPLEMENTATIONS_END
```
I found some `.mm` files doing this (i.e. not putting the `*END` macro right after definition opening brace `{`, but all the way after the definition closing brace `}`, incl. when the function body is long. It avoids indentation warning (which might be false positive) around the opening brace `{`.

dusek commented on Source/WebCore/accessibility/win/AccessibilityObjectWrapperWin.h:
```diff
@@ -30,26 +30,30 @@
 
 namespace WebCore {
 
-    class AccessibilityObject;
```
This is all just de-indenting contents inside a `namespace`, except one more change called out below re member initialization.

dusek commented on Source/WebCore/accessibility/win/AccessibilityObjectWrapperWin.h:
```diff
@@ -30,26 +30,30 @@
+    { }
+    AccessibilityObjectWrapper()
+        : m_object(nullptr)
+    { }
```
The above two constructors are the only other change in this file – changing constructor definition with member initializers from one-line to multiline, so that [member initialization has proper code style](https://webkit.org/code-style-guidelines/#punctuation-member-init).

dusek commented on Source/WebCore/accessibility/AccessibilityObject.cpp:
```diff
@@ -3691,9 +3690,9 @@ void AccessibilityObject::scrollToMakeVisibleWithSubFocus(IntRect&& subfocus) co
-         scrollParent = scrollParent->parentObject()) { }
+    for (scrollableArea = nullptr; scrollParent
+        && !(scrollableArea = scrollParent->getScrollableAreaIfScrollable());
+        scrollParent = scrollParent->parentObject()) { }
```
I think the other alternative was to write the whole `for` in a single line. Please let me know if you prefer that.

dusek commented on Source/WebCore/accessibility/AXLogger.cpp:
```diff
@@ -604,7 +604,8 @@ TextStream& operator<<(TextStream& stream, AXNotification notification)
         break;
-    WEBCORE_AXNOTIFICATION_KEYS(WEBCORE_LOG_AXNOTIFICATION)
+
+        WEBCORE_AXNOTIFICATION_KEYS(WEBCORE_LOG_AXNOTIFICATION)
```
This is borderline – `check-webkit-style` things this is something else than `case` / `default`, so expects it to be indented 4 spaces w.r.t. `switch` statement. However the macro expands to start with `case...`, so it "kind of" does start with `case` 😄.

I still obliged to get rid of the warning (and added extra newline before for better readability so that it does not visually blend with the macro definition as its another line). The other option was to put `// NOLINT` at the end of the `switch(...) {` line to silence the warning (it's not possible to put this comment on the line where the macro is used). Please let me know if you prefer that.

dusek commented on Source/WebCore/accessibility/AXTextMarker.cpp:
```diff
@@ -1488,25 +1488,27 @@ AXTextMarkerRange AXTextMarker::lineRange(LineRangeType type, IncludeTrailingLin
+        startMarker = startMarker.previousLineStart();
+
+    auto endMarker = startMarker.nextLineEnd(includeTrailingLineBreak);
+    return { WTFMove(startMarker), WTFMove(endMarker) };
```
Other option was to rewrite this all (entire diff in this file) into a `switch(type) {...}` statement with 3 cases, but I went for the above as it was a more minimal edit for now. Please let me know though if you would prefer me to re-do this as a `switch(type)`.

dusek commented on Source/WebCore/accessibility/AXObjectCache.cpp:
```diff
@@ -4218,10 +4209,10 @@ static char32_t characterForCharacterOffset(const CharacterOffset& characterOffs
+        IGNORE_CLANG_WARNINGS_BEGIN("conditional-uninitialized")
         U16_NEXT(characterOffset.node->textContent(), offset, characterOffset.node->textContent().length(), ch);
-IGNORE_CLANG_WARNINGS_END
+        IGNORE_CLANG_WARNINGS_END
```
I forgot to check if ignoring this warning is still needed before running `git-webkit pr`, so doing now but unfortunately `build-webkit` is now rebuilding the entire codebase again 🤷😠. So I will know in about 35 minutes? 😄

(If I find it's not needed, then I will amend to remove the `BEGIN`/`END` macros in the first place. If it's still needed, then this change resolves indentation warning.)

dusek replied:
OK, macros to ignore the warning still needed – without them, this happens:

```
/path/to/WebKit/Source/WebCore/accessibility/AXObjectCache.cpp:4214:9: error: variable '__c2' may be uninitialized when used here [-Werror,-Wconditional-uninitialized]
 4214 |         U16_NEXT(characterOffset.node->textContent(), offset, characterOffset.node->textContent().length(), ch);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

```

twilco commented on Source/WebCore/accessibility/AXObjectCache.h:
```diff
@@ -1058,10 +1058,9 @@ inline void AXObjectCache::setForceDeferredSpellChecking(bool shouldForce)
+class AXAttributeCacheEnabler final {
 public:
-    explicit AXAttributeCacheEnabler(AXObjectCache *cache);
+    explicit AXAttributeCacheEnabler(AXObjectCache *);
```
To avoid conflict with https://github.com/WebKit/WebKit/pull/49218/files, I'd recommend removing this part of the change. I think @minorninth fixed this when he moved `AXAttributeCacheEnabler` to its own header 👍

dusek replied:
Now done. Indeed [that PR](https://github.com/WebKit/WebKit/pull/49218/files) has fixes for both of these (i.e. the class definition opening brace to be on the same line as `class`, and removing the `cache` parameter name).

twilco commented on Source/WebCore/accessibility/AccessibilityObject.cpp:
```diff
@@ -3691,9 +3690,9 @@ void AccessibilityObject::scrollToMakeVisibleWithSubFocus(IntRect&& subfocus) co
+        && !(scrollableArea = scrollParent->getScrollableAreaIfScrollable());
+        scrollParent = scrollParent->parentObject()) { }
     if (!scrollableArea)
         return;
```
Could we re-write this entirely? This feels like an overall unintuitive use of a for-loop. I'd go for this personally:

```cpp
ScrollableArea* scrollableArea = nullptr;
while (scrollParent) {
    scrollableArea = scrollParent->getScrollableAreaIfScrollable();
    if (scrollableArea)
        break;
    scrollParent = scrollParent->parentObject();
}
```

Longer, but so much easier to understand what's happening.

dusek replied:
Done.

dusek committed:
[AX]: check-webkit-style - resolve (almost) all remaining warnings
https://bugs.webkit.org/show_bug.cgi?id=297230

Reviewed by Tyler Wilcock.

Reduces number of warnings in accessibility implementation to just 12:
$ Tools/Scripts/check-webkit-style Source/WebCore/accessibility
[...]
Total errors found: 12 in 152 files

Further 2 will be fixed as part of https://github.com/WebKit/WebKit/pull/49218
(and respective changes were removed from this commit just to avoid conflict).

The remaining 10 warnings feel like (or I found they actually are) bugs
in check-webkit-style (or code style documentation)  which I will report
as such (and through that might learn if they indeed are or if
accessibility code needs to be further fixed).

The PR (linked in bug) tries to provide diff comments for changes that
might not be obvious.

* Source/WebCore/accessibility/AXGeometryManager.cpp:
* Source/WebCore/accessibility/AXLogger.cpp:
(WebCore::operator<<):
* Source/WebCore/accessibility/AXObjectCache.cpp:
(WebCore::AXObjectCache::handleAttributeChange):
(WebCore::AXObjectCache::characterOffsetForNodeAndOffset):
(WebCore::AXObjectCache::textMarkerDataForVisiblePosition):
(WebCore::characterForCharacterOffset):
* Source/WebCore/accessibility/AXTextMarker.cpp:
(WebCore::AXTextMarker::lineRange const):
* Source/WebCore/accessibility/AXTextRun.cpp:
* Source/WebCore/accessibility/AccessibilityAttachment.cpp:
* Source/WebCore/accessibility/AccessibilityMathMLElement.h:
* Source/WebCore/accessibility/AccessibilityMockObject.cpp:
* Source/WebCore/accessibility/AccessibilityMockObject.h:
* Source/WebCore/accessibility/AccessibilityObject.cpp:
(WebCore::AccessibilityObject::leftLineVisiblePositionRange const):
(WebCore::AccessibilityObject::rightLineVisiblePositionRange const):
(WebCore::AccessibilityObject::scrollToMakeVisibleWithSubFocus const):
(WebCore::AccessibilityObject::scrollToGlobalPoint const):
(WebCore::AccessibilityObject::isWithinHiddenWebArea const):
(WebCore::AccessibilityObject::isIgnoredWithoutCache const):
* Source/WebCore/accessibility/AccessibilityProgressIndicator.cpp:
* Source/WebCore/accessibility/AccessibilityRenderObject.cpp:
(WebCore::AccessibilityRenderObject::anchorElement const):
* Source/WebCore/accessibility/AccessibilitySVGObject.cpp:
(WebCore::AccessibilitySVGObject::parentObject const):
* Source/WebCore/accessibility/AccessibilityTable.cpp:
* Source/WebCore/accessibility/AccessibilityTableRow.cpp:
(WebCore::AccessibilityTableRow::addChildren):
* Source/WebCore/accessibility/AccessibleSetValueEvent.cpp:
* Source/WebCore/accessibility/AccessibleSetValueEvent.h:
* Source/WebCore/accessibility/ForcedAccessibilityValue.h:
* Source/WebCore/accessibility/atspi/AccessibilityObjectCollectionAtspi.cpp:
(WebCore::AccessibilityObjectAtspi::matches):
* Source/WebCore/accessibility/ios/AXObjectCacheIOS.mm:
* Source/WebCore/accessibility/ios/AccessibilityObjectIOS.mm:
* Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.h:
* Source/WebCore/accessibility/ios/WebAccessibilityObjectWrapperIOS.mm:
* Source/WebCore/accessibility/isolatedtree/AXIsolatedTree.cpp:
(WebCore::AXIsolatedTree::applyPendingChangesLocked):
* Source/WebCore/accessibility/isolatedtree/mac/AXIsolatedObjectMac.mm:
* Source/WebCore/accessibility/mac/AXObjectCacheMac.mm:
(WebCore::exerciseIsIgnored):
* Source/WebCore/accessibility/mac/AccessibilityObjectMac.mm:
(WebCore::AccessibilityObject::convertRectToPlatformSpace const):
(WebCore::AccessibilityObject::subrolePlatformString const):
* Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperBase.mm:
(convertPathToScreenSpaceFunction):
(-[WebAccessibilityObjectWrapperBase accessibilityVisibleCharacterRange]):
(-[WebAccessibilityObjectWrapperBase lineRectsAndText]):
* Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperMac.mm:
(-[WebAccessibilityObjectWrapper accessibilityActionNames]):
(-[WebAccessibilityObjectWrapper accessibilityAttributeNames]):
(-[WebAccessibilityObjectWrapper windowElement:]):
(-[WebAccessibilityObjectWrapper accessibilityAttributeValue:]):
(-[WebAccessibilityObjectWrapper _accessibilityHitTest:returnPlatformElements:]):
(-[WebAccessibilityObjectWrapper accessibilityIsAttributeSettable:]):
(-[WebAccessibilityObjectWrapper accessibilityIsIgnored]):
(-[WebAccessibilityObjectWrapper accessibilityParameterizedAttributeNames]):
(-[WebAccessibilityObjectWrapper accessibilityPerformAction:]):
(-[WebAccessibilityObjectWrapper accessibilitySetValue:forAttribute:]):
(-[WebAccessibilityObjectWrapper _accessibilitySetValue:forAttribute:]):
(-[WebAccessibilityObjectWrapper accessibilityActionDescription:]):
(-[WebAccessibilityObjectWrapper accessibilityAttributeValue:forParameter:]):
(-[WebAccessibilityObjectWrapper ALLOW_DEPRECATED_IMPLEMENTATIONS_END]): Deleted.
* Source/WebCore/accessibility/win/AccessibilityObjectWrapperWin.h:
(WebCore::AccessibilityObjectWrapper::attached const):
(WebCore::AccessibilityObjectWrapper::accessibilityObject const):
(WebCore::AccessibilityObjectWrapper::AccessibilityObjectWrapper):

Canonical link: https://commits.webkit.org/298549@main
